sdk/cosmosdb/cosmos/samples/BulkUpdateWithSproc.ts

```diff
@@ -3,7 +3,7 @@
 import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError";
 import { CosmosClient } from "../dist";
 import { endpoint, key, database as databaseId, container as containerId } from "./Shared/config";
-import uuid from "uuid/v4";
+import { v4 as uuid } from "uuid";
```

Comment:
@southpolesteve, @zfoster: Do you think `uuid` should be updated in `cosmos/samples`?  The `package.json` under `cosmos\samples` doesn't list any dependency versions, so I'm not sure what version of `uuid` is assumed here.  If it should match the version of `uuid` used by `@azure/cosmos`, then the sample should be updated this way.

Reply:
Agreed. This is right.

Reply:
Thanks.  Please approve the PR if you approve of all the changes made under cosmos.